src/validators/digit.js

```diff
@@ -0,0 +1,20 @@
+import isNil from 'ramda/src/isNil';
+import isEmpty from 'ramda/src/isEmpty';
+
+const validate = val  => {
+  if (isNil(val)) {
+    return true;
+  }
+
+  if (isEmpty(val)){
+    return false;
+  }
+
+  const regexp = /^\d*$/g;
```

Comment:
You should move this outside the function so the regexp instance won't be created per function call